packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -204,14 +205,15 @@ const styles: ComponentStyles<ISiteManagedClasses, IDevSiteDesignSystem> = {
     site_paneTocRow: {
         display: "flex",
         flexWrap: "nowrap",
-        alignItems: "center",
```

Comment:
we should technically be doing seperate PR's for this or opening an issue and just assigning it as a good first issue for other contributors.